handlers/formcontrols/multiSelect.cfc

```diff
@@ -14,7 +15,10 @@ component {
 		var valueField       = args.valueField    ?: '';
 		var filterBy         = args.filterBy      ?: "";
 		var filterByField    = args.filterByField ?: filterBy;
-		var selectFields     = [ "id",labelField & " as label" ];
+
+		var labelRenderer    = args.labelRenderer = args.labelRenderer ?: presideObjectService.getObjectAttribute( object, "labelRenderer" );
+		var labelFields      = labelRendererService.getSelectFieldsForLabel( labelRenderer );
```

Comment:
Could this be:

```cfc
var labelRenderer = args.labelRenderer = args.labelRenderer ?: presideObjectService.getObjectAttribute( object, "labelRenderer" );
var selectFields  = labelRendererService.getSelectFieldsForLabel( labelRenderer );

ArrayAppend( selectFields, "id" );
if ( Len( valueField ) ) {
    ArrayAppend( selectFields, valueField );
}
```

One less variable + avoiding built in Lucee member functions and I think easier to read (note also the moving of the `valueField` appending from further below in the function).

Same feedback for the places further below (is there anyway that we can avoid repeat logic by wrapping in a function, i.e. `_prepareSelectFields( ... )`?

Reply:
Wrapped code in function. Didn't move `valueField` as that's only used in the `index` function

Reply:
super, thanks @qianqianlim !